### PR TITLE
CCAIA-0 DVRRecord revert createdAt

### PR DESCRIPTION
**Pull request by darrell-plant**

### CODE REVIEW

**Comment by habeeb01** [comment]
👍

File: app/uk/gov/hmrc/voapropertylinking/repositories/DVRRepository.scala [hunk 1]
```diff
@@ -87,10 +93,11 @@ case class DVRRecord(
       organisationId: Long,
       assessmentRef: Long,
       agents: Option[List[Long]],
-      createdAt: Long
+      createdAt: Option[BSONDateTime]
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/voapropertylinking/repositories/DVRRepository.scala:96]: if we make it optional, won't that lead to invalid JSON on the consumer side? I would have expected `createdAt` to be always present.

Comment (darrell-plant) [reply app/uk/gov/hmrc/voapropertylinking/repositories/DVRRepository.scala:96]: this is how it was in prod

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**